src/crawl/unicrawl/spiders/odisee_courses.py

```diff
@@ -44,22 +52,31 @@ def parse_main(response):
         course_name = response.xpath(f"{main_div}//h2/text()").get()
         course_id = response.xpath(f"{main_div}//h2/span/text()").get().strip(')').split(" (B-ODISEE-")[1]
         years = response.xpath("//div[@id='acjaar']/text()").get().strip("Academiejaar ").strip("Academic Year ")
-        teachers = response.xpath(f"{main_div}//span[contains(@class, 'Titularis')]//text()").getall()
-        teachers = [t.strip("\xa0|\xa0") for t in teachers]
-        teachers = [t for t in teachers if t != '']
-        languages = response.xpath(f"{main_div}//span[@class='taal']/text()").getall()
-        languages = [LANGUAGES_DICT[lang] for lang in languages]
 
-        # TODO: maybe a bit barbaric
-        content = cleanup(response.xpath("//div[contains(@class, 'tab_content')]//text()").getall())
+        teachers = cleanup(response.xpath(f"{main_div}//span[contains(@class, 'Titularis') "
+                                  f"or contains(@class, 'Coordinator')]").getall())
+        teachers = [t.strip("\xa0|\xa0").replace("  (coordinator) ", '').replace("  (coördinator) ", '')
+                    for t in teachers]
+        teachers = list(set([t.strip(" ") for t in teachers if t != '']))
+
+        languages = response.xpath(f"{main_div}//span[@class='taal']/text()").get()
+        languages = [LANGUAGES_DICT[lang] for lang in languages.split(", ")] if languages is not None else []
```

Comment:
`if languages is not None` peut être remplacé par `if languages` . A vous de voir ce que vous considérez le plus propre :)